components/scrollable-container/script.js

```diff
@@ -46,6 +46,42 @@ class ScrollableContainer extends BaseComponent {
         components.waitForFrames(() => this.scrollbar.scrollToPercents(this._scrollPos));
     }
 
+    /**
+     * Will scroll the container to a passed percents
+     * @param {string|number} value
+     * @returns {undefined}
+     */
+    scrollToPercents(value) {
+        value = parseFloat(value);
+        if (isNaN(value)) return console.error('The passed value to the scrollToPercents method should be number!');
+        if (value < 0) value = 0;
+        if (value > 100) value = 100;
+
+        this.scrollableContainer.scrollTop =
+            ((this.scrollableContainer.scrollHeight - this.scrollableContainer.offsetHeight) * value) / 100;
+        this.onScroll();
+    }
+
+    /**
+     * Will scroll to some element inside the scrollable container. Based on the alignment the container will scroll to the element by position it
+     * on the top, bottom or centering it inside the visible area of the scrollable container.
+     * @param {HTMLElement} element
+     * @param {string} alignment
+     * @returns {undefined}
+     */
+    scrollToElement(element, alignment = 'start') {
+        if (!this.scrollableContainer.contains(element)) return console.error('The passed element should be inside the scrollable container\'s content added with the <component-slot data-name="scrollable-content">!');
+
+        switch (alignment) {
+            case 'start': this.scrollableContainer.scrollTop = element.offsetTop; break;
+            case 'center': this.scrollableContainer.scrollTop = element.offsetHeight / 2 + element.offsetTop - this.scrollableContainer.offsetHeight / 2; break;
+            case 'end': this.scrollableContainer.scrollTop = element.offsetHeight + element.offsetTop - this.scrollableContainer.offsetHeight; break;
+            default: return console.error('The passed alignment argument is not valid. Possible values are "start", "center", "end"');
```

Comment:
Maybe set the default as start as well? And also provide the warning

Reply:
done